docs/mathematics/prime-number.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Prime Number
+slug: /mathematics/prime-number
+---
+
+## Introduction
+
+Prime numbers are the positive integers having only two factors, 1 and the integer itself.
+
+## Explanation
+
+If p is a prime, then it’s only factors are necessarily 1 and p itself. Any number which does not follow this is termed as composite numbers, which means that they can be factored into other positive integers.
+#### Quick Example : 
+The first ten primes are 2, 3, 5, 7, 11, 13, 17, 19, 23, 29.
+
+Here 2 is prime as it's factors are 1 and 2 itself. Similarly 3 and 5 are prime as their factors are 1 and 3 and 1 and 5 respectively.
+**Note:** It should be noted that 1 is a non-prime number.
+
+## Algorithm
+
+ - Input the number `num`.
+ - Initialize `Flag` to 1.
+ - Check the number of factors of `num`. If factor is found then `Flag` is 0.
+ - Check the condition: if `Flag` is 1 then `num` is `prime` else `num` is `not prime`.
+
+## Code 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+  defaultValue="c"
```

Comment:
The default value does not work. Can you test locally?

Reply:
It is working fine locally

Reply:
Can you please resolve the merge conflicts and see if it works fine then?